README.md

```diff
@@ -140,6 +140,16 @@ The `phrases` prop is an object that contains all the English language phrases c
   }),
 ```
 
+**Theming:**
```

Comment:
Sorry! One thing on the README... this doesn't really fall under the `## API` category imo as that related more to the actual props that you can pass into the components. I think we're going to be doing a lot of improvements on Theming and the styles in general in the future, so maybe we could have this as its own top-level section in the README?